### ISSUE TITLE
missing default for qvalue in description of Accept-Encoding (Trac #115)

### ISSUE THREAD

**Issue by mnot** [httpwg/httpbis-issues#115]
In Part 3, Section 6.3, we do not state what the default qvalue is.

I'd recommend that we state the default just once in the section that 
introduces qvalues (3.4) and remove the description of the default everywhere else.

Migrated from https://trac.ietf.org/ticket/115
```json
{
    "status": "closed",
    "changetime": "2012-03-22T00:09:07",
    "_ts": "1332374947814163",
    "description": "In Part 3, Section 6.3, we do not state what the default qvalue is.\n\nI'd recommend that we state the default just once in the section that \nintroduces qvalues (3.4) and remove the description of the default everywhere else.\n",
    "reporter": "julian.reschke@gmx.de",
    "cc": "",
    "resolution": "fixed",
    "time": "2008-05-07T13:32:33",
    "component": "p3-payload",
    "summary": "missing default for qvalue in description of Accept-Encoding",
    "priority": "normal",
    "keywords": "",
    "milestone": "unassigned",
    "owner": "julian.reschke@gmx.de",
    "type": "editorial",
    "severity": "Active WG Document"
}
```

**Comment by mnot** [comment]
Trac update at 20080519T14:43:59: 
* julian.reschke@gmx.de changed attachment from "" to "i115.diff"
* julian.reschke@gmx.de commented:

> proposed change for part 3

**Comment by mnot** [comment]
Trac update at 20080520T09:37:23: 
* julian.reschke@gmx.de commented:

> Fixed in [248]:
> 
> Resolve #115: add qvalue defaulting for Accept-Encoding (closes #115).


* julian.reschke@gmx.de changed resolution from "" to "fixed"
* julian.reschke@gmx.de changed status from "new" to "closed"

**Comment by mnot** [comment]
Trac update at 20120322T00:09:07: 
* mnot@pobox.com changed priority from "" to "normal"
* mnot@pobox.com changed severity from "Candidate WG Document" to "Active WG Document"
